engine/src/test/java/io/zeebe/engine/processor/workflow/WorkflowAssert.java

```diff
@@ -39,7 +39,7 @@ public static void assertWorkflowInstanceRecord(
       String elementId,
       Record<WorkflowInstanceRecordValue> workflowInstanceCanceledEvent) {
     Assertions.assertThat(workflowInstanceCanceledEvent.getValue())
-        .hasBpmnProcessId(PROCESS_ID)
+        .hasBpmnProcessId(elementId)
```

Comment:
Why this change? I realize this method is only used by the method above but this seems strange

Reply:
This was a left over, sorry. See my latest commit I removed this class